source/options-storage.ts

```diff
@@ -74,7 +74,8 @@ async function initializeAllOptions(): Promise<void> {
 	// Remove old domains
 	browser.permissions.onRemoved!.addListener(({origins}) => {
 		if (origins) {
-			const optionKeysToRemove = origins.map(origin => getStorageName(new URL(origin).host));
+			const optionKeysToRemove = origins.map(origin => getStorageName(new URL(origin).host))
+				.filter(key => key !== 'options');
```

Comment:
```suggestion
			const optionKeysToRemove = origins
				.filter(origin => origin !== 'https://github.com')
				.map(origin => getStorageName(new URL(origin).host));
```

Reply:
It doesn't - the actual origin would be `https://github.com/*` but still, it's not the only domain for which `getStorageName` can return `options` (`https://api.github.com/*`, `https://gist.github.com/*`)

Reply:
```suggestion
			const optionKeysToRemove = origins
				.map(origin => getStorageName(new URL(origin).host))
				.filter(key => key !== 'options');
```